src/contract.rs

```diff
@@ -354,6 +349,8 @@ fn execute_match(
             let scope = ProvenanceQuerier::new(&deps.querier).get_scope(scope_address)?;
 
             messages.push(write_scope(
+                // TODO: Use:
+                // replace_scope_owner(scope, bid_order.owner, true, None)?,
```

Comment:
The other open PR is for testing the match stuff, which should be the final piece of the puzzle after this change.  Once this is uncommented, the match code will swap ownership in its entirety to the bidder, and life will be good.  Didn't add this directly here because I figured it would mess with that other PR